pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -541,10 +541,10 @@ func (m *kubeGenericRuntimeManager) readLastStringFromContainerLogs(path string)
 }
 
 func (m *kubeGenericRuntimeManager) convertToKubeContainerStatus(status *runtimeapi.ContainerStatus) (cStatus *kubecontainer.Status) {
-	cStatus = toKubeContainerStatus(status, m.runtimeName)
+	annotatedInfo := getContainerInfoFromAnnotations(status.Annotations)
+	cStatus = toKubeContainerStatus(status, m.runtimeName, annotatedInfo)
```

Comment:
Would it be more clear to return `annotatedInfo` from `toKubeContainerStatus`? You wouldn't need to call `getContainerInfoFromAnnotations` every time you need `annotatedInfo`, it will be called once in the `toKubeContainerStatus`.